### ISSUE TITLE
How to manage more than one work spaces C9 On VPS

### ISSUE THREAD

**Issue by rinodung** [c9/core#214]
I have just installed c9 on my vps, How i can manage(create, launch) my workspaces look like c9.io?

http://my-domain:8080/ide.html

**Comment by javruben** [comment]
Did you know you can connect to your VPS using c9.io? Check out this article on SSH workspaces: https://docs.c9.io/docs/running-your-own-ssh-workspace

Would that resolve your issue?

**Comment by rinodung** [comment]
Yes I see your solution, But I want to manage my workspaces(ram, cpu) by installing c9 on my VPS. But I do not know how to do that?

**Comment by javruben** [comment]
The standalone version allows you to do that for yourself. It doesn't come with a way to manage your environments, but I'm not sure you'll need that.

If you need this for more than 1 person, please email sales@c9.io for more information.

**Comment by orditeck** [comment]
Maybe you already know that, if not that could help: you need to specify your workspace (in other words the project folder) while starting `server.js`, so you can do `nodejs server.js -w /home/myuser/myproject/ --listen 0.0.0.0 --auth test:test` and you'll have a workspace for that project.

That being said, it doesn't seem like there is a way right now to manage your workspace with an UI. I'm not sure something like that will be made open to public from c9 neither. I guess it will have to come from the community.

I installed c9/core last week and I'd like to keep everything on my server, and have multiple workspaces that I can manage with an UI. If I wouldn't already be doing a personal project right now, I'd create a simple UI allowing to create/start/stop workspaces. It's actually very simple.

**Comment by rinodung** [comment]
@javruben Thanks for your response. I want to deploy C9 for my university, I think C9 is a perfect open source project that I'm looking for... I do not know whether I have permission to do that?

@orditeck Exactly what I want, You mean that I have to do it myself without any document support from C9, right? May you show me your project(link) that manage your multiple workspaces with UI. Thanks.

**Comment by orditeck** [comment]
@rinodung I actually said that if I wouldn't already be doing another personal project right now, I'd create a simple UI allowing to create/start/stop workspaces. So when I wrote those line, there were no UI for that.

However, I just finished it for fun. You can [access it here: orditeck/c9ui on github](https://github.com/orditeck/c9ui)

![c9ui](http://i.imgur.com/BSbwbfh.jpg)

**Comment by rinodung** [comment]
Wow! It's wonderful project by Lumen Framework. I will follow your project and study how you do that? Thanks @orditeck so much

**Comment by knkrth** [comment]
You can try https://github.com/Taisun-Docker/taisun.git
